app/models/review_issue.rb

```diff
@@ -86,8 +86,8 @@ def editor
     body.match(/\*\*Editor:\*\*\s*(@\S*|Pending)/i)[1]
   end
 
-  # Extract the Reviewer from the issue body
-  def reviewer
-    body.match(/\*\*Reviewer:\*\*\s*(@\S*|Pending)/i)[1]
+  # Extract the Reviewers from the issue body
+  def reviewers
```

Comment:
This just matches the logic in [WhedonAPI](https://github.com/openjournals/whedon-api/blob/b0720d8c30673b87c5bb8860d361b403338b31e9/whedon.rb#L266-L268)